victor-nweze-deel/StackOverflow-lite
# #160040075 Add docs to REST API

#### What does this PR do?
* Add SwaggerUI customized documentation to REST API

#### Description of Task to be completed?
* Create a Swagger.json file containing REST endpoints
* Power documentation using SwaggerUI

#### How should this be manually tested?
* Visit [nweze-stackoverflow.herokuapp.com/api-docs](https://nweze-stackoverflow.herokuapp.com/api-docs) to check the documentation

#### Any background context you want to provide?
* None

#### What are the relevant pivotal tracker stories?
* [#160040075](https://www.pivotaltracker.com/n/projects/2189268)

#### Screenshots (if appropriate)
![swaggerdoc](https://user-images.githubusercontent.com/41582565/44622632-2f8ab800-a8b4-11e8-8a0b-01af66d86794.png)


#### Questions:
* None
